ddmrp_history/models/stock_warehouse_orderpoint.py

```diff
@@ -218,21 +230,23 @@ def stacked(df, categories):
             p = figure(
                 x_range=(dates[0], dates[-1]), y_range=(start_stack, top_y),
                 x_axis_type='datetime')
+            p.sizing_mode = 'stretch_both'
+            p.toolbar.logo = None
 
             p.grid.minor_grid_line_color = '#eeeeee'
             p.patches([x2] * len(areas), [areas[cat] for cat in categories],
                       color=EXECUTION_COLORS, alpha=0.8, line_color=None)
+            date_format = self._get_lang_date_format()
```

Comment:
You can remove _get_lang_date_format and use this:
```suggestion
            date_format = self.env['res.lang']._lang_get(self.env.lang).date_format
```
Or self.env.user.lang

Reply:
Thanks!